.github/workflows-helpers/run-e2e-test-job-template.yaml

```diff
@@ -49,7 +49,11 @@ spec:
               value: "false" # Flip to true to see the command and result of the execution
             - name: POKTROLLD_HOME
               value: /root/.poktroll
-            - name: PATH_HOST
+            - name: PATH_URL
+              value: http://${NAMESPACE}-path:3000/v1
+              # As we can't expose many DNS names, but the test requires to send requests to different subdomains (e.g. anvil., rest., etc.),
```

Comment:
Can you update this comment. I read it a few times still don't understand the root source of the issue.